btcjson/v2/btcjson/btcdextcmds.go

```diff
@@ -58,6 +58,19 @@ func NewDebugLevelCmd(levelSpec string) *DebugLevelCmd {
 	}
 }
 
+// GenerateCmd defines the generate JSON-RPC command.
+type GenerateCmd struct {
+	NumBlocks uint32 `json:"numblocks"`
```

Comment:
No need to specify the tag on the command.  JSON-RPC 1.0 uses positional params.